packages/frontend/src/components/nft/NFTDetail.js

```diff
@@ -0,0 +1,225 @@
+
+const StyledContainer = styled(Container)`
+    .container {
+        margin-left: auto;
+        margin-right: auto;
+        max-width: 429px;
+        position: relative;
+
+        img {
+            width: 100%;
+            max-width: 429px;
+            margin-bottom: 83px;
+
+            filter: drop-shadow(0px 100px 80px rgba(0, 0, 0, 0.07)) 
+            drop-shadow(0px 41.7776px 33.4221px rgba(0, 0, 0, 0.0503198)) 
+            drop-shadow(0px 22.3363px 17.869px rgba(0, 0, 0, 0.0417275)) 
+            drop-shadow(0px 12.5216px 10.0172px rgba(0, 0, 0, 0.035)) 
+            drop-shadow(0px 6.6501px 5.32008px rgba(0, 0, 0, 0.0282725)) 
+            drop-shadow(0px 2.76726px 2.21381px rgba(0, 0, 0, 0.0196802));
+        }
+
+        .desc {
+            margin-top: 48px;
+            margin-bottom: 56px;
+
+            font-family: Inter;
+            font-style: normal;
+            font-weight: 500;
+            font-size: 16px;
+            line-height: 150%;
+            /* or 24px */
+
+            display: flex;
+            align-items: center;
+
+            /* gray/neutral/800 */
+
+            color: #272729;
+        }
+    }
+
+    .owner {
+        p {
+            font-family: Inter;
+            font-style: normal;
+            font-weight: 500;
+            font-size: 12px;
+            line-height: 150%;
+            /* identical to box height, or 18px */
+
+            display: flex;
+            align-items: center;
+            letter-spacing: 0.115em;
+
+            /* gray/neutral/500 */
+
+            color: #A2A2A8;
+        }
+
+        .inner {
+            height: 30px;          
+            line-height: 30px; 
+            display: flex;         
+            align-items: center;
+
+            span {
+                font-family: Inter;
+                font-style: normal;
+                font-weight: 500;
+                font-size: 16px;
+                line-height: 150%;
+                /* identical to box height, or 24px */
+
+                display: flex;
+                align-items: center;
+
+                /* gray/neutral/800 */
+
+                color: #272729;
+            }
+        }
+    }
+
+    .transfer-btn {
+        width: 100%;
+        margin-top: 56px !important;
+
+        svg {
+            margin-right: 10px !important;
+        }
+    }
+
+    #back-btn {
+        position: absolute;
+        left: -98px;
+        width: 30px !important;
+        height: 30px !important;
+        
+        svg {
+            width: 100%;
+            height: 100%;
+        }
+    }
+
+`;
+
+const UserIcon = styled.div`
+    background-size: 21px;
+    flex: 0 0 40px;
+    height: 40px;
+    border-radius: 50%;
+    background-color: #f8f8f8;
+    text-align: center;
+    margin: 0 12px 0 0;
+    
+    svg {
+        width: 26px;
+        height: 26px;
+        margin: 7px;
+    }
+
+    @media (min-width: 940px) {
+        display: inline-block;
+    }
+`;
+
+function arrowSVG () {
```

Comment:
We can remove this once we use the `<BackArrowButton>`

Reply:
oh actually this is for the arrow icon on transfer button.

Reply:
I see. Please use the existing [`<SendIcon/>`](https://github.com/near/near-wallet/blob/master/packages/frontend/src/components/wallet/Wallet.js#L383) then and adjust style if needed